psi4/driver/procrouting/dft_funcs/dict_dh_funcs.py

```diff
@@ -0,0 +1,577 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2017 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+"""
+List of double-hybrid functionals
+"""
+
+funcs = []
+
+funcs.append({
+    "name": "B2PLYP",
+    "x_functionals": {
+        "GGA_X_B88": {
+            "alpha": 0.47
+        }
+    },
+    "x_hf": {
+        "alpha": 0.53
+    },
+    "c_functionals": {
+        "GGA_C_LYP": {
+            "alpha": 0.73
+        }
+    },
+    "c_mp2": {
+        "alpha": 0.27
+    },
+    "citation":
+    '    S. Grimme, J. Chem. Phys., 124, 034108, 2006\n',
+    "description":
+    '    B2PLYP Double Hybrid Exchange-Correlation Functional\n',
+})# yapf: disable
```

Comment:
I've added `yapf: disable`'s to the dicts so they remain formatted somewhat nicely.